### PR TITLE
[14.0][MIG] module_analysis: Migration to 14.0

### PR DESCRIPTION
**Pull request by santostelmo**

### CODE REVIEW

File: module_analysis/models/ir_module_module.py [hunk 1]
```diff
@@ -155,8 +153,8 @@ def button_analyse_code(self):
 
             # Update the module with the datas
             values = {}
-            for file_ext, analyses in analysed_datas.items():
-                for k, v in analyses.items():
+            for _, analyses in analysed_datas.items():
```
Comment (legalsylvain) [inline module_analysis/models/ir_module_module.py]: Hi. I know that it is a python convention (to use underscore) however, it is confusing in odoo project where _ is used for translation.

Comment (pedrobaeza) [reply module_analysis/models/ir_module_module.py]: You can use as well any variable prefixed with underscore (for example `for _disposable, analyses in ...`). Anyway, in this case I would use instead:
```suggestion
            for analyses in analysed_datas.values():
```

Comment (santostelmo) [reply module_analysis/models/ir_module_module.py]: Hi, indeed the usage of the variable _ can be avoided.

**Review by atchuthan — APPROVED** [review]
Could squash last 3 commits to [MIG] module_analysis: Migration to 14.0 for clean history
Otherwise looks good :+1:

**Review by nbessi — APPROVED** [review]

**Comment by dreispt** [comment]
Please fix precommit and conflicts

**Comment by matiasperalta1** [comment]
@santostelmo Ping. Could you fix precommit and conflicts?

**Comment by etobella** [comment]
@santostelmo Some test failed, can you rebase?

**Comment by thomaspaulb** [comment]
Superseded by https://github.com/OCA/server-tools/pull/2656

**Pull request closed without merging**